library/Zend/InputFilter/InputInterface.php

```diff
@@ -26,6 +26,13 @@ public function setAllowEmpty($allowEmpty);
      */
     public function setBreakOnFailure($breakOnFailure);
 
+
+    /**
+     * @param bool $continueIfEmpty
+     * @return self
+     */
+    public function setContinueIfEmpty($continueIfEmpty);
+
```

Comment:
We use this method @ [Input](https://github.com/zendframework/zf2/blob/develop/library/Zend/InputFilter/Input.php#L300) , i think we have to add something to support the old `InputInterface` too, maybe with `method_exists`, or you know a better way?